dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/controller/ProcessDefinitionController.java

```diff
@@ -245,42 +234,40 @@ public Result updateProcessDefinition(@ApiIgnore @RequestAttribute(value = Const
                                           @RequestParam(value = "description", required = false) String description,
                                           @RequestParam(value = "releaseState", required = false, defaultValue = "OFFLINE") ReleaseState releaseState) {
 
-        Map<String, Object> result = processDefinitionService.updateProcessDefinition(loginUser, projectName, id, name,
-            processDefinitionJson, description, locations, connects);
+        Map<String, Object> result = processDefinitionService.updateProcessDefinition(AuthUtils.getAuthUser(), projectName, id, name,
+                processDefinitionJson, description, locations, connects);
         //  If the update fails, the result will be returned directly
         if (result.get(Constants.STATUS) != Status.SUCCESS) {
             return returnDataList(result);
         }
 
         //  Judge whether to go online after editing,0 means offline, 1 means online
         if (releaseState == ReleaseState.ONLINE) {
-            result = processDefinitionService.releaseProcessDefinition(loginUser, projectName, id, releaseState);
+            result = processDefinitionService.releaseProcessDefinition(AuthUtils.getAuthUser(), projectName, id, releaseState);
         }
         return returnDataList(result);
     }
 
     /**
      * query process definition version paging list info
      *
-     * @param loginUser           login user info
-     * @param projectName         the process definition project name
-     * @param pageNo              the process definition version list current page number
-     * @param pageSize            the process definition version list page size
+     * @param projectName the process definition project name
```

Comment:
It is best not to modify the existing format, you need to take a check

Reply:
recover this delete